src/librustc/mir/cache.rs

```diff
@@ -220,10 +220,6 @@ impl ReadOnlyBodyCache<'a, 'tcx> {
         self.cache.unwrap_predecessor_locations(loc, self.body)
     }
 
-    pub fn body(&self) -> &'a Body<'tcx> {
```

Comment:
@eddyb Thank you so much for pointing on the `&'a` in the `Deref` impl. That was the only reason this fn was left around, and it helped clean up a lot of code. CC @oli-obk